com.onlyoffice.DesktopEditors.appdata.xml

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<application>
+  <id type="desktop">com.onlyoffice.DesktopEditors</id>
+  <name>ONLYOFFICE Desktop Editors</name>
+  <developer_name>ONLYOFFICE</developer_name>
+  <summary>Office productivity suite</summary>
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>LicenseRef-AGPL-3.0</project_license>
+  <url type="homepage">https://onlyoffice.com</url>
+  <icon type="remote" height="256" width="256">https://res.cloudinary.com/canonical/image/fetch/f_auto,q_auto,fl_sanitize,w_72,h_72/https://dashboard.snapcraft.io/site_media/appmedia/2018/01/icon_IzCN8xA.png</icon>
```

Comment:
This is...using snapcraft's image URL?

Reply:
There was no icons available so I grabbed from Snapcraft at the moment. It shouldn't be an issue.